app/modules/settings/views/templates/display_settings.hbs

```diff
@@ -0,0 +1,35 @@
+<h2 class="layout-title">{{I18n 'display'}}</h2>
```

Comment:
this appears as `afficher` in French, while that should probably be `affichage`, but as there is no distinction in English, we need to use a trick to make the distinction, which is to set different keys. In the past, we have been using `x_verb` (ex: `search_verb`), so either we could replace previous occurrences of `display` by `display_verb` or replace that one by `display_name`